src/plusplus.coffee

```diff
@@ -142,7 +142,7 @@ module.exports = (robot) ->
     reasons = scoreKeeper.reasonsForUser(name)
 
     reasonString = if typeof reasons == 'object' && Object.keys(reasons).length > 0
-                     "#{name} has #{score} points. here are some raisins:" +
```

Comment:
I don't really know the story behind it, but the author of the original factoids bot calls "++ reasons" raisins.